### ISSUE TITLE
Globus Connect between curation station and PRDS workstations

### ISSUE THREAD

**Issue by kelynch** [pulibrary/princeton_ansible#3153]
Ensure that connections between PDC Describe Globus endpoints mounted to curation station and PRDS workstations are established and tested.

Contingent on the setup of the curation station (currently in progress)

## References

* [Globus Connect - Personal](https://www.globus.org/globus-connect-personal)

**Comment by kelynch** [comment]
This ticket is no longer relevant.
